resources/How_to_make_your_bot_competitive.md

```diff
@@ -4,18 +4,10 @@
 
 Typically only one bot-operator is able to close each arbitrage opportunity, making bot operation competitive in nature. The purpose of this document is to provide ideas on ways to improve your own bot. 
 
-### Gas Priority Fee Optimization
-In config/network.py there is a constant: DEFAULT_GAS_PRICE_OFFSET. This is a multiplier that increases the priority fee paid by your TX. The higher it is, the more likely your TX will get executed, but the more the TX costs. It’s set to increase by 9% by default, which is likely insufficient to be competitive. You could also design a custom function to modify the priority fee based on profit - this could likely make you competitive, but would require some work. 
```

Comment:
This section should be updated - not removed - as priority fee optimization is still an important part of competitiveness.

Reply:
The increase-factor has been removed, because it doesn't have a meaningful functionality.
See bullet 1 in the description of this PR for more details.
Subsequently, this description becomes irrelevant.

Reply:
The description became irrelevant, but the topic is still extremely relevant. 

Can we leave it with something like:
Creating custom handling for the bot's max priority fee can increase your bot's competitiveness.

Reply:
I agree with your analysis regarding adding a more configurable gas strategy, however, it's not necessary for this PR. 

My main concern here is informing bot-operators that this is something they can customize to be more competitive. 

Even a simple sentence that informs them of this factor is sufficient for now from my perspective.

Reply:
Updated; please review